esmvaltool/recipes/examples/recipe_julia.yml

```diff
@@ -5,6 +5,8 @@ documentation:
   description: |
     Recipe for example diagnostic written in Julia.
 
+  title: Recipe running an example diagnostic written in Julia Roberts.
```

Comment:
While I appreciate the work of Julia Roberts, perhaps Julia Child may be more appropriate for testing recipes. On second thought you probably just mean Julia, the language.

Reply:
cheers, I'll exclude Julia Roberts from the cast :laughing: